homepage.html

```diff
@@ -136,7 +136,7 @@ <h3>In the Loop</h3>
                 {% if next_meeting %}
 
                 {% if next_meeting.description %}
-                <p itemprop="description">{{ next_meeting.description|truncatewords:80 }}</p>
+                <p itemprop="description">{{ next_meeting.description|truncatewords:80|safe }}</p>
```

Comment:
I don't think you want to use safe here. This would render the first few starter HTML tags but then wreck the rest of the page, right?

Reply:
Seems to render ok, cannot crash it.  Can take safe out but then the p tags show.  seems like main meetings don't have a description anyway though.  

is there an extra homepage.html file in addition to one under templates? when I remove it, I don't see any problems.   

seeing opposite Javascript behavior for the [more]/[less] functionality under This Month's Topic section with |safe, could not figure that out 
<img width="655" alt="Screen Shot 2019-10-24 at 11 52 28 AM" src="https://user-images.githubusercontent.com/11667547/67507622-e7505c80-f654-11e9-861e-1a49b37a1db4.png">